### ISSUE TITLE
Steam Overlay and Fedoraware menu conflicting

### ISSUE THREAD

**Issue by km-ch** [M-FeD/Fedoraware#105]
**Fedoraware Version Date: 10/18/21**  

## Config used:

N/A

## Description:  

If you shift + tab while you have your Fedoraware menu open, it will display above Steam's overlay and you can't do anything, forcefully having to close the game

## Steps to reproduce:  
- [x] I am able to consistently reproduce this error

1. Open TF2
2. Inject Fedora~~paste~~ware
3. Press your menu key (insert)
4. Do Shift + Tab to open the Steam Overlay
5. magic
[EXTRA] 6. Try to interact with Fedoraware's menu and bam, you're stuck

## Additional context 

N/A

## Screenshot of cheats enabled  

N/A

- [x] I have included the steps to reproduce to the best I am able to share.

**Comment by JAGNEmk** [comment]
Yep, imgui being imgui, might selfcode a surface menu soon tho!!!!!!!
